test/e2e/route_advertisements.go

```diff
@@ -837,9 +837,11 @@ var _ = ginkgo.DescribeTableSubtree("BGP: isolation between advertised networks"
 				func(ipFamilyIndex int) (clientName string, clientNamespace string, dst string, expectedOutput string, expectErr bool) {
```

Comment:
I think I can easily add a kapi server test to ensure that part works

Reply:
Good idea! lets do that :)

Reply:
done